picard/webservice/__init__.py

```diff
@@ -97,60 +97,95 @@ def __init__(self, response_type):
 class WSRequest(QNetworkRequest):
     """Represents a single HTTP request."""
 
-    def __init__(self, method, host, port, path, handler, parse_response_type=None, data=None,
-                 mblogin=False, cacheloadcontrol=None, refresh=False,
-                 queryargs=None, priority=False, important=False,
-                 request_mimetype=None):
+    def __init__(
+        self,
+        *,
+        method=None,
+        host=None,
+        port=-1,
+        path=None,
+        handler=None,
+        parse_response_type=None,
+        data=None,
+        mblogin=False,
+        cacheloadcontrol=None,
+        refresh=False,
+        queryargs=None,
+        priority=False,
+        important=False,
+        request_mimetype=None,
+    ):
         """
         Args:
             method: HTTP method.  One of ``GET``, ``POST``, ``PUT``, or ``DELETE``.
             host: Hostname.
             port: TCP port number (80 or 443).
             path: Path component.
             handler: Callback which takes a 3-tuple of `(str:document,
-            QNetworkReply:reply, QNetworkReply.Error:error)`.
+                QNetworkReply:reply, QNetworkReply.Error:error)`.
             parse_response_type: Specifies that request either sends or accepts
-            data as ``application/{{response_mimetype}}``.
+                data as ``application/{{response_mimetype}}``.
             data: Data to include with ``PUT`` or ``POST`` requests.
             mblogin: Hints that this request should be tied to a MusicBrainz
             account, requiring that we obtain an OAuth token first.
             cacheloadcontrol: See `QNetworkRequest.Attribute.CacheLoadControlAttribute`.
             refresh: Indicates a user-specified resource refresh, such as when
-            the user wishes to reload a release.  Marks the request as high priority
-            and disables caching.
+                the user wishes to reload a release.  Marks the request as high priority
+                and disables caching.
             queryargs: `dict` of query arguments.
-            retries: Current retry attempt number.
```

Comment:
I removed this one about `retries` since this is a leftover from old changes, it makes no sense in the current code.